### DISCUSSION TITLE
Link buttons color bug

### DISCUSSION THREAD

**Discussion by mendax1234** [pradyunsg/furo#466] [Potential Issue] [answered]
### What's happening?

In the dark mode , the link button's color is strange.
![image](https://user-images.githubusercontent.com/62934294/176119150-2225d3a4-1112-4993-b164-c458de08bd3a.png)
In the light mode , it should be looked like this.
![image](https://user-images.githubusercontent.com/62934294/176119219-d13a25d7-3c18-4528-845b-7c013cfa8861.png)

### Reproducer

https://6940swerve-docs.readthedocs.io/en/latest/
You can have a look at the website itself and the github repo is also linked to it.

### Expectation

l expect the whole link buton to be dark except the words.(At least the background of the link button should be dark l think)

Hope this can be fixed!

Best
Daniel

### Code of Conduct

- [X] I agree to follow the [Code of Conduct](https://github.com/pradyunsg/furo/blob/main/CODE_OF_CONDUCT.md).

**Comment by pradyunsg** [comment]
Thus isn't a bug in Furo, but coming from whatever tool you're using to generate the cards.

That tool's markup is not aware of the fact that Furo is in dark mode. You'll either need to add custom CSS or use the only-light/only-dark mechanism to get the colors right.

**Comment by mendax1234** [comment]
@pradyunsg 
l find that the grid may fix the issue but how can l use grid table?l think l have know is syntax but l don't know what should l install if l want to use grid table, can you give me some suggesstions about it? Thanks!

**Comment by pradyunsg** [comment]
I'm not sure what you're using to create these cards but I'd recommend reaching out to the maintainers of that project for help. I'm not the right person to help you with this sadly.

**Reply by mendax1234** [reply]
> I'm not sure what you're using to create these cards but I'd recommend reaching out to the maintainers of that project for help. I'm not the right person to help you with this sadly.

![image](https://user-images.githubusercontent.com/62934294/176127535-1c2ac44c-55ab-4214-865d-e6e9c877ffde.png)
I use these lines to create a block , but it just goes wrong.

All right.

**Comment by pradyunsg** [comment, marked as answer]
Looks like this has been resolved? The link shown has a page that looks good in dark mode and light mode.
